Include/scripts/groovy/com/kms/katalon/core/reporting/basic/reporting/JsStepModel.java

```diff
@@ -173,14 +175,26 @@ private void initLogRecords() {
                             attachmentFile = new File(logFolder + File.separator + messageLog.getAttachment());
                         }
                     }
+                    
                     if (attachmentFile.exists()) {
                         try {
-                            jsLogRecModel.props.add(new JsModelProperty("link",
-                                    "data:image/png;base64," + encodeFileContent(attachmentFile), listStrings));
+                            String projectDir = RunConfiguration.getProjectDir();
+                            BundleSettingStore bundleSettingStore = new BundleSettingStore(projectDir, "com.katalon.plugin.report", true);
+                            boolean useBase64 = !bundleSettingStore.getBoolean("generateHTMLReferenceImages", false);
+
+                            String linkValue;
+                            if (useBase64) {
+                            	linkValue = "data:image/png;base64," + encodeFileContent(attachmentFile);
+                            } else if (attachmentFile.getPath().startsWith("/var/folders/")) {
+                                linkValue = attachmentFile.getName();
+                            } else {
+                                linkValue = attachmentFile.getPath();
```

Comment:
Should always use relative path so you should keep the original value from the log record above.